LEAF_Request_Portal/sources/Form.php

```diff
@@ -1782,9 +1784,7 @@ public function hasDependencyAccess($dependencyID, $details)
 
                     return $this->checkIfBackup($empUID);
                  }
-
-
-                break;
+                 // unreachable code so no break
```

Comment:
Maybe the whole case should be commented out if it's not reachable.

Reply:
the break was the only thing not reachable. in the if statement just before this there is a return statement in both the true and false sections. So you would never get to the break. I put the comment here so that in the future when we fix these files and only have one return per method we wouldn't forget to put the break back in.